docs/blog/2025-Q2-community-board.rst

```diff
@@ -0,0 +1,67 @@
+.. post:: May 30, 2025
+   :tags: community board, news, report
+   :author: Mikey Ariel
+
+Write the Docs Community Board Quarterly Update Q2-2025
+=======================================================
+
+Greetings, documentarians! Mikey Ariel here, chairperson of the WTD community board. I'm happy to share our second quarterly update for 2025 with you. 
+
+To catch up on previous quarterly updates, check out the `community board blog posts <https://www.writethedocs.org/blog/archive/tag/community-board/>`_ archive. 
+
+Q2-2025 quarterly meeting highlights
+------------------------------------
+
+Our second meeting of the year focused mainly on the newly-announced `Write the Docs Berlin 2025 <https://www.writethedocs.org/conf/berlin/2025/>`_ conference, which is our first in-person official conference in Europe since the pandemic. 
+
+We're grateful for the virtual conference teams for all of the hard work they've done over the past few years, and look forward to combining the in-person and virtual experience so that our documentarians can choose to attend in the way that best fits their needs. 
+
+Here are the highlights from the meeting:
+
+**Next stop - Berlin**
+    Over the past few months, the board has been disucssing the possibility of returning to in-person conferences in Europe. Ever since we moved to virtual conferences, we've been monitoring the community and industry trends in Europe as well as attendance data and feedback on our virtual conferences (first for Portland and later on for Atlantic). 
```

Comment:
```suggestion
    Over the past few months, the board has been discussing the possibility of returning to in-person conferences in Europe. Ever since we moved to virtual conferences, we've been monitoring the community and industry trends in Europe as well as attendance data and feedback on our virtual conferences (first for Portland and later on for Atlantic). 
```